rosa_planning/rosa-cloud-expert-prereq-checklist.adoc

```diff
@@ -157,6 +157,16 @@ Prerequisites needed from a networking standpoint.
 
 * Configure your firewall to allow access to the domains and ports listed in xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs[AWS firewall prerequisites].
 
+=== Additional custom security groups
+
+When you create a cluster using an existing VPC, you have the option to add additional custom security groups during cluster creation. Complete these prerequisites before you create the cluster:
```

Comment:
Same comments for these wordings as the other file where you had similar wording.

However, it looks like it is slightly different. You could consider whether these could be put into a module and included both places

Reply:
Slightly different on purpose unfortunately (one is a very quick reference checklist, one is a detailed requirements list), but I've made the wording changes.